pkg/detectors/aha/aha.go

```diff
@@ -12,16 +12,19 @@ import (
 	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
 )
 
-type Scanner struct{}
+type Scanner struct {
+	client *http.Client
+}
 
 // Ensure the Scanner satisfies the interface at compile time.
 var _ detectors.Detector = (*Scanner)(nil)
 
 var (
-	client = common.SaneHttpClient()
+	defaultClient = common.SaneHttpClient()
 
 	// Make sure that your group is surrounded in boundary characters such as below to reduce false positives.
 	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"aha"}) + `\b([0-9a-f]{64})\b`)
+	URLPat = regexp.MustCompile(`\b([A-Za-z0-9](?:[A-Za-z0-9\-]{0,61}[A-Za-z0-9])\.aha\.io)`)
```

Comment:
how did you discover this (and other) regex changes?

Reply:
i was trying to fix the test by generating new tokens. realized that the url we used to verify this was the generic `https://company.aha.io/api/v1/me` from the docs, and not the actual subdomain that we needed (no the requests don't work with the generic one). so i added this to capture the url as well (a format we use for other detectors that require URLs).

i'll add comments on the other regexes i updated separately.